client/tinycbor/cbor.h

```diff
@@ -0,0 +1,606 @@
+/****************************************************************************
+**
+** Copyright (C) 2017 Intel Corporation
```

Comment:
Which kind of license is this? A stripped down BSD license?

Reply:
https://github.com/intel/tinycbor/blob/master/LICENSE
MIT License

Reply:
Permissions:
 Commercial use
 Modification
 Distribution
 Private use